hhvm-serve.go

```diff
@@ -1,10 +1,10 @@
 package main
 
-import "github.com/beberlei/hhvm-serve/fcgiclient"
+import "github.com/LionsAd/hhvm-serve/fcgiclient"
```

Comment:
Couldn't we just do the following instead?

    import "github.com/mholt/caddy/middleware/fastcgi/fcgiclient"

I searched for a Github hosted version of the code before, if there is one now we could just use it instead.